virttest/env_process.py

```diff
@@ -716,6 +774,14 @@ def preprocess(test, params, env):
     logging.debug("KVM version: %s" % kvm_version)
     test.write_test_keyval({"kvm_version": kvm_version})
 
+    # Checking required kernel, if not satisfied, cancel test
```

Comment:
Checked kvm module info on RHEL7 and fedora, cannot find the detail version for kvm. Only can get the version of kernel. So I think we could drop the kvm version here.
$ modinfo kvm
filename:       /lib/modules/4.11.8-300.fc26.x86_64/kernel/arch/x86/kvm/kvm.ko.xz
license:        GPL
author:         Qumranet
depends:        irqbypass
intree:         Y
vermagic:       4.11.8-300.fc26.x86_64 SMP mod_unload 
signat:         PKCS#7
signer:         
sig_key:        
sig_hashalgo:   md4
parm:           allow_unsafe_assigned_interrupts:Enable device assignment on platforms without interrupt remapping support. (bool)
parm:           ignore_msrs:bool
parm:           min_timer_period_us:uint
parm:           kvmclock_periodic_sync:bool
parm:           tsc_tolerance_ppm:uint
parm:           lapic_timer_advance_ns:uint
parm:           vector_hashing:bool
parm:           halt_poll_ns:uint
parm:           halt_poll_ns_grow:uint
parm:           halt_poll_ns_shrink:uint